tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -153,6 +195,18 @@ void RunInference(Settings* s) {
     LOG(INFO) << "number of outputs: " << outputs.size() << "\n";
   }
 
+#if defined(__ANDROID__)
```

Comment:
Please see the other comment about removing these `if` blocks. If the platform isn't Android and `GetDelegates` returns an empty map, the code block below should still work.